examples/minio-tls/thanos-object-storage.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: v1
+data:
```

Comment:
for example secret, can we use "stringData", but not to use "data" to hide all details?

Reply:
Yes. that makes more sense to show the content of secret directly.